webapp/nuxt.config.js

```diff
@@ -62,7 +62,8 @@ export default {
       },
       {
         name: 'viewport',
-        content: 'width=device-width, initial-scale=1',
+        // content: 'width=device-width, initial-scale=1', // "width=device-width" does not work correctly for Safari on iOS, see https://stackoverflow.com/a/37136777
```

Comment:
Git history has this stored, please remove comment.

Reply:
Done